m4_sessions/1804inning/public_data.md

```diff
@@ -5,12 +5,56 @@ Anna Royer
 Sabrina Kennedy
 Jared Eklin
 
-Name of Session: 
+#### Name of Session: Using Public Data for Civic Tech
 
-Session Description (Why should students come to this session):
+#### Session Description (Why should students come to this session):
+Discussing and Engaging with material and others on what civic tech and public data are and how to use them to solve problems irl.
 
-Session Outcomes (What will students learn/leave with?):
+#### Session Outcomes (What will students learn/leave with?):
+* Students will be emboldened to become involved in Civic Tech projects
+* Students will know what Civic Tech is and how it is being used or has been used in the past
+* Students will have concrete resources to becoming involved in Civic Tech
+* Students will brainstorm ideas of ways to become involved in Civic Tech
+ 
+#### Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+**Ice Breakers** (7 minutes)
+TBD
 
-Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+**Intro** (10 minutes)
+* What is Civic Tech?
+* Real examples of how people have used and are using public data for civic tech
+* What classifies data as "public"
+* What is an API? 
+
+**Open Questions**(5 minutes)
+ 
+**Breakout** (7 minutes)
+(4 small groups: each group discusses one of the following questions)
```

Comment:
This is great. Let me know if you want chart paper and markers for each group to chart their discussion

Reply:
Chart paper and markers would be great for this discussion. Thank you!